Sources/AsyncHTTPClient/HTTPHandler.swift

```diff
@@ -836,6 +846,7 @@ extension TaskHandler: ChannelDuplexHandler {
                 }
 
                 return promise.futureResult.map {
+                    self.actualBodyLength += part.readableBytes
```

Comment:
- assertion about the correct EL missing
- test that would fail this assertion right now probably missing.

Reply:
moved the increment to where it is certainly on correct EL